SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -269,6 +269,18 @@ bool FLocalDeploymentManager::LocalDeploymentPreRunChecks()
 		}
 	}
 
+	if (!IsServiceRunningAndInCorrectDirectory()) 
```

Comment:
Will this return false if the server isn't running at all?

Reply:
@m-samiec It's a good catch, let me double check this.